### DISCUSSION TITLE
[Question] Will new code search replace the old elasticsearch based search in future?

### DISCUSSION THREAD

**Discussion by fawazahmed0** [community/community#19332] [Other Feature Feedback, Questions, & Ideas] [answered]
I am asking because, sometimes I find elasticsearch based search more accurate.

**Comment by look** [comment, marked as answer]
@fawazahmed0 yes, our plan is to replace the existing Elasticsearch-based code search. It would be really helpful for us if you can give examples of queries that give you better results in the current code search, so we can improve the new search.

One reason why you may not be finding what you're looking for with the new search is that it hasn't indexed as many repositories as the current search. That will change as we scale up.

**Reply by fawazahmed0** [reply]
Thanks, I tried searching again, the results are almost identical. Is there any file size limit for new code search. The old elastic search has file size [limit of 384kb](https://docs.github.com/en/search-github/searching-on-github/searching-code#:~:text=only%20files%20smaller%20than%20384%20kb%20are%20searchable.)

**Reply by look** [reply]
The new system's limit is currently similar. We may increase this in the future, but we're currently focused on adding more repositories rather than supporting extremely large files.

**Reply by fawazahmed0** [reply]
Thanks
